languages/csharp/exercises/concept/attack-of-the-trolls/AttackOfTheTrollsTests.cs

```diff
@@ -3,6 +3,30 @@
 public class PermissionsTests
 {
     [Fact]
+    public void Permission_none_is_an_enum_field()
```

Comment:
Maybe rephrase slightly?

```suggestion
    public void None_permission_is_defined()
```